plugins/module_utils/network/sonic/config/l3_interfaces/l3_interfaces.py

```diff
@@ -183,9 +183,10 @@ def _state_replaced(self, want, have):
         """
         ret_requests = list()
         commands = list()
-        new_want = self.update_object(want)
-        new_have = self.remove_default_entries(have, False)
-        get_replace_interfaces_list = self.get_interface_object_for_replaced(new_have, want)
+        new_have = remove_empties_from_list(have)
+        new_have = self.remove_default_entries(new_have)
+        new_want = self.remove_default_entries(want)
```

Comment:
Please ensure empties are removed in 'want' to prevent empty values from generating diff.
```suggestion
        new_want = remove_empties_from_list(want)
        new_want = self.remove_default_entries(new_want)
```

Reply:
Thank you for noticing this. I have committed your suggestion.